aiven/client/client.py

```diff
@@ -445,6 +445,55 @@ def restart_kafka_connector_task(self, project, service, connector_name, task_id
         url = "/project/{}/service/{}/connectors/{}/tasks/{}/restart"
         return self.verify(self.post, url.format(project, service, connector_name, task_id))
 
+    def get_schema(self, project, service, schema_id):
+        url = "/project/{}/service/{}/kafka/schema/schemas/ids/{}"
+        return self.verify(self.get, url.format(project, service, schema_id))
+
+    def check_schema_compatibility(self, project, service, subject_name, version, schema):
+        url = "/project/{}/service/{}/kafka/schema/compatibility/subjects/{}/versions/{}"
+        return self.verify(self.post, url.format(project, service, subject_name, version), body={"schema": schema})
+
+    def get_schema_global_configuration(self, project, service):
+        return self.verify(self.get, "/project/{}/service/{}/kafka/schema/config".format(project, service))
+
+    def update_schema_global_configuration(self, project, service, compatibility):
+        url = "/project/{}/service/{}/kafka/schema/config"
+        return self.verify(self.put, url.format(project, service), body={"compatibility": compatibility})
+
+    def get_schema_subject_configuration(self, project, service, subject_name):
+        return self.verify(self.get, "/project/{}/service/{}/kafka/schema/config/{}".format(project, service, subject_name))
+
+    def update_schema_subject_configuration(self, project, service, subject_name, compatibility):
+        url = "/project/{}/service/{}/kafka/schema/config/{}"
+        return self.verify(self.put, url.format(project, service, subject_name), body={"compatibility": compatibility})
+
+    def list_schema_subjects(self, project, service):
+        return self.verify(self.get, "/project/{}/service/{}/kafka/schema/subjects".format(project, service))
+
+    def delete_schema_subject(self, project, service, subject_name):
+        url = "/project/{}/service/{}/kafka/schema/subjects/{}"
+        return self.verify(self.delete, url.format(project, service, subject_name))
+
+    def get_schema_subject_version(self, project, service, subject_name, version):
+        url = "/project/{}/service/{}/kafka/schema/subjects/{}/versions/{}"
+        return self.verify(self.get, url.format(project, service, subject_name, version))
+
+    def get_schema_subject_version_schema(self, project, service, subject_name, version):
+        url = "/project/{}/service/{}/kafka/schema/subjects/{}/versions/{}/schema"
+        return self.verify(self.get, url.format(project, service, subject_name, version))
```

Comment:
Subject names can presumably also have unsafe characters and should be quoted. Also I noticed that a couple of the Kafka Connect related APIs don't do quoting for connector name. Maybe adding some method like

```
@staticmethod
def build_path(*parts):
    return "/" + "/".join(quote(part, safe="") for part in parts)
```

and always using that to build paths like

```
path = self.build_path("project", project, "service", service, "kafka", "schema", "subjects", subject_name, "versions", version, "schema)
```

would make sense.

Reply:
Added `build_path` in PR #98.